### DISCUSSION TITLE
Who wants to be hired (Feb 2023)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#45532] [Show and tell]
# What is this?
This discussion is for Next.js developers looking for work.

# Candidates

Share your information if you are looking for work. Please use this format:

  Location:
  Remote:
  Willing to relocate:
  Technologies:
  Resume/CV:
  Website:
  Email:

Good luck with interviewing!

---

If you got hired through this job board, please reach out to me (hassan@vercel.com) so I can send you some swag :)

If you're a company looking to post a role, use #45533.

*If you're looking for last month's discussion, see #44540. We've had yet another [*success story*](https://twitter.com/nutlope/status/1610667757565083648) of a developer being hired!*

**Comment by edwardsmoses** [comment]
Hello, 
I am Edwards, a dev with solid experience in React and Next.js with TypeScript. With 8 years of development, I've experience working with early-stage startups helping launch web products from ground-up. 


Remote: Yes
Willing to relocate: No
Technologies: React, Next.js, TypeScript, React Native, Ionic Framework, Node.JS, .NET Code, GraphQL

Website: [edwardsmoses.com](https://www.edwardsmoses.com)
Resume/CV: [LinkedIn profile](https://www.linkedin.com/in/edwards-moses/)
Email: [hi@edwardsmoses.com](mailto:hi@edwardsmoses.com)

**Comment by rammyblog** [comment]
Hello,
I am Tunde, a full-stack dev with solid experience in Django, NodeJS, Express, Svelte and ReactJS. With 4 years of development,  Bachelor’s Degree in Computer Engineering. I am also an active open-source community member, with several projects under my belt. This has helped me stay up to date with the latest technologies and developments in the field. I believe it will be a valuable asset to your team.

Remote: Yes
Willing to relocate: Yes
Technologies: Python, Django, Express, React, Next.js, TypeScript, Node.JS.

Website: [tundeonasanya.dev](https://tundeonasanya.dev)
Resume/CV: [Resume](https://docs.google.com/document/d/1v8eCsG_eLnftjjCVV5AZCtbNPYAnWBz_g0cT3bibA-8/edit)
Email: [onasanyatunde67@gmail.com](mailto:onasanyatunde67@gmail.com)

**Comment by brunofrigeri** [comment]
I’m Bruno, Web and Mobile Developer with 4 years of experience with a focus on React, Next.js, React Native, GraphQL,
and Relay. I’m passionate about technology, travel, reading and writing. I’m also a self-learner and can work well
under pressure.

Remote: Yes
Willing to relocate: Yes

**Technologies:**
React.js, React Native, Next.js, Typescript, Javascript, GraphQL, Apollo, React Relay, Firebase, AWS, Webpack, HTML, CSS,

Vercel, Github Actions, Azure, Buildkite, Codeflow, Bitrise.

LinkedIn: [bruno-frigeri](https://www.linkedin.com/in/bruno-frigeri/)
Twitter: [brunofrigerip](https://twitter.com/brunofrigerip)
Resume/CV: [Resume](https://drive.google.com/file/d/1s51TjnRGX45HLLfivug66KbvaV_MevoF/view?usp=share_link)
Website: [brunofrigeri.com](https://www.brunofrigeri.com/)
Email: bpiraja97@gmail.com

Thanks! Feel free to check my github or contact me via email

**Comment by masnormen** [comment]
Howdy! 👋

I have experience working in both big startup (Indonesia’s _#1 ecommerce marketplace) and small-scale software house. I'm open and curious to explore other fields as well!

Frontend Web is my strongest skill, but I'm mostly a tech generalist with a passion and affinity for digging deep into different discipline and topics. Outside of work, I also love open source. I have created some npm libraries and contributed to some open-source projects in the past as well.

Currently I am open for full-time or contract positions in local or global teams.

**Location:** Indonesia (GMT+7)
**Remote:** Yes
**Willing to relocate:** Yes
**Technologies:** JavaScript, TypeScript, Rust, Next.js❤️, React, Svelte, Node.js, Tailwind CSS, Material UI, Jenkins, Postgres, Shell script, UNIX ecosystem, REST, Redis, GraphQL, Cloudflare Workers, Supabase

**Resume:** https://nourman.id/resume
**Website:** https://nourman.id
**Email:** nourmanhajar+work@gmail.com
**Linkedin:** https://www.linkedin.com/in/nourmanhajar/

Thank you.

**Comment by FlynnFc** [comment]
Hey! 👋

I began learning to code in 2019. Since then, I fell in love with Frontend development and kickstarted my self-taught Journey. You can see my personal projects alongside client-commissioned projects in my [portfolio](https://flynnfc.dev/). I am incredibly motivated, and a self starter. I'm brilliant at solving problems with little external support. I mainly focus on Frontend (JavaScript, TypeScript, React and Next) but my programming and web development knowledge is agnostic. I have a strong understanding of mobile-first development and accessibility.

The project Im most proud of is **Learn with chat**
- https://learnwithchat.com/

The react made dashboard was made to help a streamer with a segment during their broadcast in which viewers post educational youtube videos in chat. the site checks for any duplicate links then it calls the youtube video & data API to retrieve data such as thumbnails, channel, views, video length etc. The streamer then can choose a video from a clear and informative dashboard rather than blindly clicking links.


- Decreased stream downtime from 40 minutes to 8
- Increased viewer retention by 30%
- you can see more details here: https://flynnfc.dev/getsmartersaturday

I am open to all full-time Frontend or Full-stack opportunities.

Location: London (UK)
Remote: Yes
Willing to relocate: Yes
Technologies: JavaScript, TypeScript, Next.js, React, Tailwind, Prisma, MySQL, REST, Redis, Supabase, PlanetScale, T3 Stack.

Resume: [link](https://wosipkxcwhwqrtnbwdxx.supabase.co/storage/v1/object/public/cv/FlynnFoleyClarkeCV.pdf)
Website: https://flynnfc.dev/
Email: [foleyclarkef@gmail.com](mailto:foleyclarkef@gmail.com)
Linkedin: https://www.linkedin.com/in/flynnfc/

Thank you for taking the time to read my post.

**Comment by cuevaio** [comment]
Hi there 👋! 
Anthony here, self-taught, quick learner, and passionate about frontend engineering.
I'm eager to take on my first junior role and continue to grow as a developer.

Location: Lima, Peru
Remote: `true`
Willing to relocate: `true`

Technologies: `TypeScript` `React` `Next.js` `Vercel` `Tailwind CSS` `FaunaDB` `Xata`
Resume/CV: [cuevantn.com/cv](https://www.cuevantn.com/cv)

Website: [cuevantn.com](https://www.cuevantn.com)
Email: hi.cuevantn@gmail.com

**Comment by FredySandoval** [comment]
```txt 

   .-.                      
  /'v'\                     My name is Fredy, +8 years of experience,                                __
 (/   \)                    Full Stack Developer, UX/UI Designer, 3D Illustrator,                  <(o )___
==="="===<                  Google Cloud Admin expert, English/Spanish speaker.                     ( ._> /
   |_|                                                                                               `---' 
```
|||||||||||||||||||||
|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|



Remote: Yes
Website: [fredy.dev](https://www.fredy.dev/)
Email: me [at] fredy.dev
|||||||||||||||||||||
|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|

**Reply by imaPheven** [reply]
i like the minimalist approach. :D

**Comment by cond0r** [comment]
🖐️ Hi, my name is Catalin, and I've been a Full-Stack Developer for more than ten years. I have considerable expertise in Typescript, Tailwind CSS, React, Next.js, Node.js, GraphQL, Prisma, tRPC. E-commerce enthusiast, and contributor to [Next.js Commerce](https://github.com/vercel/commerce). I'm looking forward to building something great together 🚀

Location: Romania
Remote: Yes
Willing to relocate: No
Resume/CV: [LinkedIn](https://www.linkedin.com/in/catalin-pinte-088b0922/)
Email: [pintecatalin@gmail.com](mailto:pintecatalin@gmail.com)

**Comment by zhyd1997** [comment]
Hi 👋 there

I am Yadong Zhang from China and am a frontend developer, also a freelancer.

Currently just looking for a **full-time** job, let me introduce myself more:

After graduated in July 2019 with Bachelor's degree (no CS background), I started my CS journey for a year, and then got my first job at [**Compass**](https://github.com/UrbanCompass): in March 2021, but just working for a few months, that was a good experience in the listed company.

In the second half of the year, of course, I applied many job opportunities (Including ByteDance and RedHat) but unfortunately, all of them failed, then I found a part-time job at [**Stargately**](https://github.com/stargately/ ) (web-boommo app, which is private in GitHub), I learned that every developer should have his/her own product so I launched my own side project called [**SoftMaple**](https://github.com/softmaple/softmaple), which is a paper typesetting editor for professors and students. The project is promoted by Github in Dec 2021: https://github.blog/2022-01-21-release-radar-dec-2021/

And in 2022, I became a freelancer at Web3 field:

Contributed to

- **Superfluid** ecosystem: (bug fixes and features)

  - [Ricochet-Exchange](https://github.com/Ricochet-Exchange/ricochet-frontend) - _Polygon_ Network

  - [Geo-Web](https://github.com/Geo-Web-Project/cadastre) - _Optimism_ Network

- **Synthetix** ecosystem: (bug fixes and features)

  - [Kwenta](https://github.com/Kwenta/kwenta) - _Optimism_ Network

- **Uniswap** (small bug fixes)

  - https://github.com/Uniswap/interface

Also traditional web2 open source project:

- **Storybook** (small bug fixes)

1. [storybook](https://github.com/storybookjs/storybook)

2. [design-system](https://github.com/storybookjs/design-system/)

3. [eslint-plugin-storybook](https://github.com/storybookjs/eslint-plugin-storybook)
4. [mdx2-csf](https://github.com/storybookjs/mdx2-csf)

- **ESLint** ecosystem (small bug fixes)

  - [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint)

Location: **China**
Remote: **Yes**
Willing to relocate: **Yes**
Technologies: **React**, **Nextjs**, **Emotionjs**, **TypeScript**
Resume/CV: https://www.linkedin.com/in/yadong-zhang-48a474154/
Website: https://github.com/zhyd1997
Email: zhyd007@gmail.com

**Comment by jainpranayr** [comment]
Hi there! I'm Pranay Jain, a highly motivated and experienced web developer with over 2 years of experience in the field. I specialize in using ReactJS, NextJS, TypeScript, and Tailwind CSS to create modern and dynamic web applications. I'm always eager to learn and take on new challenges, and I'm confident that my skills and passion for web development will make me a valuable addition to any team

Location: India
Remote: Available for remote work
Willing to relocate: Open to relocation opportunities
Technologies: ReactJS, TypeScript, NextJS, Tailwind CSS, JavaScript
Resume/CV: [link](https://drive.google.com/file/d/1qDY116Kv6kdAwymGaI6X72LlykLX4EOF/view?usp=share_link)
Email: [jpranay688@gmail.com](mailto:jpranay688@gmail.com)

**Comment by mastercodercat** [comment]
As a senior frontend developer specializing in React and Next.js with TypeScript as well as web3 technologies, Shane(me) is looking for new positions and growth opportunities. With over 9 years of experience in front-end development, I've worked as a frontend-heavy full stack developer, and participated in web/web3 projects with modern technologies and libraries, mainly with React, React Native, Next.js, web3.js/ethers.js, and Solidity.
I prefer to work on a growing team and startups with great teammates and prefer a fully-remote work environment.
I am available for full-time positions as well as part-time or contract positions in global teams.

Location: Canada
Remote: Yes
Willing to relocate: No
Technologies: **Next.js**, **React**, **React Native**, **Ionic**, **TypeScript**, **GraphQL**, **Cypress**, **RTL**, **Tailwind**, **Node.js**, **Express.js**, **Nest.js**, **Prisma**, **Solidity**, **web3.js/ethers.js**
Resume: [My Resume](https://drive.google.com/file/d/1fw3_udibFbQyhubnga_oqeejMLCgUA3n/view?usp=sharing)
Website: [Portfolio](https://shanelewis.vercel.app)
Email: shanelewix@gmail.com

**Comment by pratik-codes** [comment]
Hi there,

I am Pratik Tiwari, a software engineer with 3+ years of experience working with the React and Node.js ecosystem, as well as AWS and Ruby on Rails. I have built various projects for clients, including a university management app and fintech products. My major personal project is Dashlit, a browser extension that offers a range of productivity features similar to Momentumdash.

GitHub (https://github.com/pratik-codes) and LinkedIn (https://www.linkedin.com/in/pratiktiwari12/) profiles.


Location: India
Remote: Yes
Willing to relocate: Yes
Technologies: Reactjs, Nodejs, Typescript 
Resume/CV: https://drive.google.com/drive/u/2/folders/11fMGlmzgEQm9YisRvrbHsuqzUtcG0xt-
Website: https://tiwaripratik.com/
Email: pratiktiwari1212@gmail.com

**Comment by bgdnvk** [comment]
Hey

I'm Bogdan, a Full Stack SWE, I'm looking for a long term role, you will see on my profile/CV that I've been working with many different stacks including Java, NodeJS, Go, TS and even Ruby (RoR) or Flutter in the past. I've done React as well and right now I'm digging into NextJS, hence how I found this thread.

Location: Spain, EU.
Remote: Yes.
Willing to relocate: Yes, I have Spanish nationality so working anywhere in the EU isn't a problem. Open to other places as well.
Technologies: TypeScript, React, NextJS, NodeJS, Go, Java, Spring Boot, Python, PostgreSQL, MongoDB, and a long etc.
Website/tech blog: https://bognov.tech/
LinkedIn: https://www.linkedin.com/in/bogdan-novykov/
Resume/CV: https://bognov.tech/cv
Email: check CV or contact me on LinkedIn

Cheers

**Reply by kitinmayo** [reply]
emailed you!

**Comment by ahmadyusufcodes** [comment]
Hola!

I am a highly skilled and experienced software developer with four years of experience in developing web and mobile applications. I possess a proven ability to use technologies such as JavaScript, React, React Native, Node.js, Next.js, Vue.js, Nuxt.js, MongoDB and more to deliver high-quality software products. I am experienced in following industry best practices, writing clean and efficient code, and being a quick learner. I am seeking a challenging role where he can utilize his experience and skills to contribute to the development of innovative software products.

Location: Abuja, Nigeria.
Remote: Yes.
Willing to relocate: Yes.
Website: under development
LinkedIn: https://www.linkedin.com/in/ahmadcodes
Resume/CV: https://drive.google.com/file/d/1rbu9dUL9tPrFDNlLcgx7H2wP07Nr6hXo/view?usp=share_link
Email: [ahmadee.by@gmail.com](mailto:ahmadee.by@gmail.com)
Technologies: JavaScript, React, React Native, Node.js, Next.js, Vue.js, Nuxt.js, MongoDB, AWS, Git, Agile,
HTML, CSS.

Kampai!🥂

**Comment by jensen** [comment]
Location: Vancouver, BC
Remote: Yes
Willing to relocate: No
Technologies: Full-Stack JavaScript (Check GitHub)
Resume/CV: https://www.linkedin.com/in/karljensendev/
Website: https://github.com/jensen

**I am only looking for contract work at this time.**

**Comment by christoph724** [comment]
Hello!

My name is Christopher Martin from the US, just call me Chris.
I'm a Sr. Front-end developer with over 8 years of rich experience in web development especially on frontend side.
My key skills are React and Next.js. I can work at any timezone in the US. I'm very familiar with remote work.
Skills: React, Next.js, Vue.js, Nuxt.js, JavaScript, TypeScript, HTML5, SASS ...

Email: christophmartin724@gmail.com
_Resume_: [Indeed Resume](https://my.indeed.com/p/christopherm-m6sqx1b)
_Linkedin_: http://www.linkedin.com/in/christopher-martin-frontend-developer
_Skype_: live:.cid.f894bec847a3ed60
Phone: 7185142772

Please feel free to reach out to me at any time.

Best Regards,
Chris

**Comment by gyenabubakar** [comment]
I am a Front-end Engineer with a passion for creating visually appealing and user-friendly web applications. With a strong eye for design and attention to detail, I strive to create unique and engaging user experiences that deliver business value. I am ready to bring my skills and experience to a dynamic and challenging team environment.

**Location:** Accra, Ghana
**Remote:** Yes
**Willing to relocate:** Yes
**Technologies:** HTML, CSS, JavaScript, TypeScript, Git, React, Next.js, Vue, Nuxt.js, Svelte, SvelteKit, Jest, React Testing Library, Svelte Testing Library, vue-test-utils, Cypress, Storybook
**Resume/CV:** https://drive.google.com/file/d/1twJSvCshpg4j6YMlwHPyX7ktSbpaL0wj/view?usp=sharing
**Website:** https://blog.gyen.dev
**Email:** [gyenabubakar@gmail.com](mailto:gyenabubakar@gmail.com)

**Comment by ratracegrad** [comment]
My name is Jennifer Bland. I am a Senior Software Engineer. Google Developers Expert. Entrepreneur. Mountain Climber. Neil Diamond fanatic. MBA grad. World traveler.

**Location:** United States
**Remote:** Yes
**Willing to relocate:** No
**Technologies:** React, Next.js, Vue. HTML, CSS, JavaScript.  Jest and Cypress.  MongoDB, Supabase, SQL. Git, Agile.
**Google Developers Expert:** [https://developers.google.com/community/experts/directory/profile/profile-jennifer-bland](https://developers.google.com/community/experts/directory/profile/profile-jennifer-bland)
**Website:** [https://www.jenniferbland.com/](https://www.jenniferbland.com/)
**Resume/CV:** [https://www.jenniferbland.com/resume/](https://www.jenniferbland.com/resume/)
**Email:** [ratracegrad@gmail.com](mailto:ratracegrad@gmail.com)
**Github:** [https://github.com/ratracegrad](https://github.com/ratracegrad)
**LinkedIn:** [https://www.linkedin.com/in/ratracegrad/](https://www.linkedin.com/in/ratracegrad/)

**Comment by A-alharbi9** [comment]
My name is Abdullah, a full stack developer, I strive to create great products and design eloquent systems. Versatility, adaptability and self-learning are my strong points. My experience and passion will make your company revolutionary!

I'm looking forward to bring my talents and skills to your team

**Remote:** Yes
**Technologies:** HTML, CSS, bootstrap, TailwindCSS, JavaScript, React, Redux, Next, MongoDB, PHP and MySQL
**Email:** A21alharbi9@gmail.com

**Comment by CuriouslyCory** [comment]
Hello! I'm (Curiously) Cory Sougstad. I've been a full stack developer since 2005 working both solo and on larger teams, in small startups and fortune 500 enterprises alike. I'm a data geek and process theorycrafter, and I love solving complex problems with code! 

Location: AZ, United States
Remote: Yes
Willing to relocate: No
Technologies: Typescript, Javascript, NextJs, React, Angular, Solidity, wagmi, Redux, CSS, Tailwind, tRPC, NestJs, PostgreSQL, NodeJs, and lots more.
Resume/CV: [Resume](https://curiouslycory.com/corysougstad-resume.pdf)
Website: [https://curiouslycory.com/](https://curiouslycory.com/)
Email: [cory@curiouslycory.com](mailto:cory@curiouslycory.com)

**Comment by ndungtse** [comment]
Hello! I'm Ndungutse Charles, software developer with fullstack experience. I always try to run new thing and exploring new tech stacks. I'm also good at teamwork and communication. 

Location: Rwanda
Remote: Yes
Willing to relocate: Yes
Technologies: Next.js, React, React Native, Vuejs, TypeScript, GraphQL, Cypress, Tailwind, Node.js, Express.js, Nest.js, Prisma, SpringBoot,
Resume: [My Resume](https://drive.google.com/file/d/1OOI9y8YDmjqdckP55a9kZ9V1qbhHK0KP/view)
Website: [Portfolio](https://www.ndungutsecharles.me/)
Email: [ndungutsecharles103@gmail.com](mailto:ndungutsecharles103@gmail.com)

**Comment by imaPheven** [comment]
Been programming since 1994. Questions recruiters ask:

## What tech stacks do you know?
This list is long but let me do a short list:
> - NodeJs (ExpressJs, Mocha/Chai, Jest, TypeScript, Functional Programming in Javascript), 
> - Python (AppEngine, Torando, circus),
> - Front-end UI (Angular, React, Vue JS), 
> - Front-end Styling (Bootstrap, Material UI, Tailwind),
> - Data (Mongo, Postgres, MySql/MariaDb, Redis, Oracle, SQL Server, R, numpy), 
> - AWS (S3, SQS, SNS, DynamoDB, Lambda, EC2, cloud formation, ElastiCache, DocumentDB, Aurora, Route-53), 
> - GCP (AppEngine, BigQuery), 
> - Magento Cloud (PHP, MySql, Redis, ElasticSearch), 
> - CI/CD (Jenkins, GitHub, Docker, Teraform, Ruby, Serverless), 
> - Custom Linux builds, Bash, zsh, 
> -  Objective-C (iOS), Swift (iOS), 
> - WebRTC, SIP
> - .Net / C# MVC, WebAPI, dotnet 6+

## What tech stacks are you looking to work in
> Pretty much ANYTHING that does not require a M$ Windows environment. 
> However, I ❤️ javascript so NodeJS backend, any DB stack, and React front-end. I'm cool with any styling framework ... bootstrap, material design ... preferably tailwind.

## How much do we need to pay you?

> I'm willing to work for free if it's a fun and cool project with a cool group of people. (My wife disagree's with me on this subject)

... my youngest kid is sick. Will pick up another time.
[Youtube: One Love (Bob Marley) feat. Manu Chao | Playing For Change | Song Around The World](https://www.youtube.com/watch?v=4xjPODksI08&list=RDcXACMQtuuZI&index=3)

**Comment by jnlsn** [comment]
Self-directed Software Engineer dedicated to crafting unique, accessible, user-focused experiences; able to build dynamic web applications from start to finish and refactor legacy systems. Advocate for automation and best practices in design, performance, and testing; streamline new code and utilize strict linting rules to help sustain performance and simplify processes for future updates. Welcome the opportunity to contribute to project planning and establish a foundation within which development teams can function quickly and efficiently. Collaborate effectively with designers and other engineers in the early phases of development to avoid problems down the line. Stay apprised of cutting-edge developments in web technology. Draw on skills and experience to resolve coding issues as they arise; interested in training and coaching others to do the same.

Location: Denver, CO
Remote: ✅ 
Willing to relocate: ❌ 
Technologies: TypeScript, React, GraphQL, Next.js, HTML, CSS
Resume/CV: https://linkedin.com/in/nielsen
Website: https://jaynielsen.com
Email: nlsn at duck dot com

**Comment by wangtiger317** [comment]
I am a Senior Frontend developer and have 7 years of experience on the web and mobile apps development.
Skills: React, Next, typescript, javascript, tailwind css , Bootstrap, html, SCSS , electron.js
Remote: ✅
Willing to relocate: ❌
website: https://portofolio-pied.vercel.app
let me know

**Comment by khattakdev** [comment]
Hey,
I am a Developer Advocate; I was laid off last week from MLH. I have been in the industry since 2020 and currently looking for a job in the DevRel department. 
I have worked with companies like GitHub to [create content](https://www.twitch.tv/collections/qoCTWhSOzharJA) for their Twitch channel. Last week, one of my courses on Vitejs was featured on [FreeCodeCamp's YouTube channel](https://www.youtube.com/watch?v=VAeRhmpcWEQ). Moreover, I actively make content and help my [Instagram](https://www.instagram.com/khattakdev) audience of over 10K learn frontend technologies.

Location: ✅
Remote: ✅
Willing to relocate: Maybe
Technologies: React, Next, JavaScript, TailwindCSS
Resume/CV: [Here](https://docs.google.com/document/d/1DCXQO6zfaWZQCj-YTztL4QzLK6LBnKVZP-MQ7GYK9mY/edit?usp=sharing)
Website: [Here](https://khattak.dev/)
Email: me@khattak.dev

Following are some blogs by other companies that have featured me:
[Microsoft Student Ambassador Graduate](https://techcommunity.microsoft.com/t5/student-developer-blog/meet-a-recent-microsoft-learn-student-ambassador-graduate/ba-p/3610786) - Microsoft
[How I went from Introvert to Developer Advocate](https://stories.mlh.io/how-i-went-from-introvert-to-developer-advocate-266131d7be2c) - MLH
[MLH Top 50 of 2022](https://top.mlh.io/2022/profiles/arsalan-khattak) - MLH
![image](https://user-images.githubusercontent.com/37709578/228645463-04a54496-c202-41cc-bf01-4327bb59f302.png)
![image](https://user-images.githubusercontent.com/37709578/228645715-8dca7097-2c82-480c-933d-8489daa9f91d.png)
